CONTRIBUTING.md

```diff
@@ -91,14 +94,18 @@ You should be able to access `http://localhost:5122` and see the SigLens UI. If
 
 ### Send Data to SigLens
 
-To send data, clone the [sigscalr-client](https://github.com/sigscalr/sigscalr-client) repo.
+To send data, go to `tools/sigclient`.
 
-In another terminal, start the ingestion via `sigscalr-client` by running:
+In another terminal, go to the location of sigclient:
+```
+cd tools/sigclient
+```
```

Comment:
You can delete line 97 since you have the same instruction in line 101

Reply:
sure